fix(tt-rss) add docs and gui clean up (#6712)
truecharts/charts: charts/stable/tt-rss/docs/installation_notes.md

> # Installation Notes
> 
> - Set `TTRSS Self URL Path` to **http://scale_ip:APP_PORT/** and then login using the default [credentials](./credentials.md).
>   - If you plan to use ingress and a domain you need to do the task above and then edit the app to add ingress and set `TTRSS Self URL Path` to **https://app.mydomain.tld/**.

Reviewer: ```suggestion
# Installation Notes

- Set `TTRSS Self URL Path` to **http://scale_ip:APP_PORT/** and then login using the default [credentials](./credentials.md).
  - If you plan to use ingress and a domain you need to:
    - Set `TTRSS Self URL Path` to **https://app.mydomain.tld/** 
    - Configure ingress on the app
```

>   - If you plan to use ingress and a domain you need to do the task above. Turn the app **off** and then edit the app to add ingress and set `TTRSS Self URL Path` to **https://app.mydomain.tld/** and then turn the app **on** afterwards.

Reviewer: Don't the app starts when you save the changes?

Author: i tested it a few times, i seen it deploy and stop if the initial state is stopped, but I found just leaving the app off to apply the changes works more than applying the changes live.

Reviewer: That's because of a known bug. Should be fixed in the new common
